WordPress/WordPressTest/BloggingPromptsServiceTests.swift

```diff
@@ -403,31 +373,27 @@ private extension BloggingPromptsServiceTests {
         return try XCTUnwrap(passedParameters[key], "Param not found for \(key)")
     }
 
-    func passedNumber() throws -> Int? {
-        return try passedParameter("per_page") as? Int
+    func passedNumber() throws -> Int {
```

Comment:
Just a nit – if we're `throw`-ing anyway, we can ensure that the test data will always be present